distribution/src/main/assembly/dist.xml

```diff
@@ -246,7 +246,7 @@
             </excludes>
         </fileSet>
         <fileSet>
-            <directory>../documentation/manual/target/docbook/publish</directory>
+            <directory>../documentation/manual/target/asciidoctor</directory>
```

Comment:
Instead of  `asciidoctor`, can we call it `output` or `asciidoctor-output` or something similar?

Not important obviously, I just find it harder to figure out what the folder contains from the name.